File: docs/examples/anomaly_true_and_eccentric.py
import pytest
import rastro

if __name__ == '__main__':
    nu = 45.0; # Starting true anomaly
    e = 0.01;  # Eccentricity

    # Convert to eccentric anomaly
    ecc_anomaly = rastro.anomaly_true_to_eccentric(nu, e, True)

    # Convert back from eccentric to true anomaly
    nu_2 = rastro.anomaly_eccentric_to_true(ecc_anomaly, e, True)

    # Confirm equality to within tolerance
    assert nu == pytest.approx(nu_2, abs=1e-14)

File: docs/examples/anomaly_true_and_eccentric.rs
use rastro::orbits::{anomaly_true_to_eccentric, anomaly_eccentric_to_true};

fn main() {
    let nu = 45.0; // Starting true anomaly
    let e = 0.01;  // Eccentricity

    // Convert to eccentric anomaly
    let ecc_anomaly = anomaly_true_to_eccentric(nu, e, true);
    println!("Rust Hello World {}", ecc_anomaly);
}